worldedit-core/src/main/java/com/fastasyncworldedit/core/extension/platform/binding/ProvideBindings.java

```diff
@@ -84,7 +84,7 @@ public RollbackDatabase database(World world) {
     }
 
     public Region[] regions(Player player, FaweMaskManager.MaskType type) {
```

Comment:
Javadocs

Reply:
It's a common method between the two above. Should be private. Quite obvious what it does as well